packages/core/src/amazonq/webview/ui/quickActions/generator.ts

```diff
@@ -28,7 +28,7 @@ export class QuickActionGenerator {
                         ? [
                               {
                                   command: '/dev',
-                                  placeholder: 'Briefly describe a task or issue',
+                                  placeholder: 'Describe your task or issue in as much detail as possible',
```

Comment:
non-blocker: Should we store this text in a constant and refer to it from all 3 places we need?

Reply:
I don't see any of the messages being shared for webview ui across, as all messages are explicitly mentioned within the generators.